src/Microsoft.DocAsCode.Build.RestApi/Swagger/Internals/SwaggerJsonBuilder.cs

```diff
@@ -18,13 +18,17 @@ namespace Microsoft.DocAsCode.Build.RestApi.Swagger.Internals
     internal class SwaggerJsonBuilder
```

Comment:
>SwaggerJsonBuilder [](start = 19, length = 18)

is there any unit test around this?

Reply:
SwaggerJsonParserTest#ParseSwaggerJsonWithLoopReferenceShouldSucceed() with hit the cache